flux-dsl/src/main/java/com/influxdb/query/dsl/functions/RangeFlux.java

```diff
@@ -149,4 +161,17 @@ public RangeFlux withStop(@Nonnull final String stop) {
 
         return this;
     }
+
+    /**
+     * @param stop Specifies the exclusive newest time (Unix timestamp in seconds) to be included in the results
+     * @return this
+     */
+    @Nonnull
+    public RangeFlux withStop(final long stop) {
```

Comment:
Please use `Long` object:

```suggestion
    public RangeFlux withStop(final Long stop) {
```